[2022-08-02 14:27:40,028] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: pokea.respond 2022-08-02T14:10:00+00:00 [queued]>
[2022-08-02 14:27:40,060] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: pokea.respond 2022-08-02T14:10:00+00:00 [queued]>
[2022-08-02 14:27:40,061] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 14:27:40,061] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2022-08-02 14:27:40,062] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2022-08-02 14:27:40,078] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): respond> on 2022-08-02T14:10:00+00:00
[2022-08-02 14:27:40,082] {standard_task_runner.py:51} INFO - Started process 2178 to run task
[2022-08-02 14:27:40,089] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'pokea', 'respond', '2022-08-02T14:10:00+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pokea.py', '--cfg-path', '/tmp/tmpykzg356n']
[2022-08-02 14:27:40,093] {standard_task_runner.py:76} INFO - Job 9: Subtask respond
[2022-08-02 14:27:40,172] {logging_mixin.py:103} INFO - Running <TaskInstance: pokea.respond 2022-08-02T14:10:00+00:00 [running]> on host b4dcda255553
[2022-08-02 14:27:40,256] {taskinstance.py:1232} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=pokea
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-08-02T14:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-02T14:10:00+00:00
[2022-08-02 14:27:40,257] {python.py:118} INFO - Done. Returned value was: Greet Responded Again
[2022-08-02 14:27:40,302] {taskinstance.py:1142} INFO - Marking task as SUCCESS. dag_id=pokea, task_id=respond, execution_date=20220802T141000, start_date=20220802T142740, end_date=20220802T142740
[2022-08-02 14:27:40,347] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-02 14:27:40,379] {local_task_job.py:118} INFO - Task exited with return code 0
